typify-cli/src/main.rs

```diff
@@ -0,0 +1,29 @@
+use schemars::schema::Schema;
+use std::env;
+use typify::TypeSpace;
+
+fn main() {
+    let args: Vec<String> = env::args().collect();
+    if args.len() != 2 {
+        panic!("{} need 1 argument", env!("CARGO_PKG_NAME"))
```

Comment:
perhaps a gentler failure mode and usage message?

Reply:
done